.github/workflows/CI.yml

```diff
@@ -26,14 +27,16 @@ jobs:
           python-version: "${{ matrix.python-version }}"
       - name: Setup Fortran
         uses: fortran-lang/setup-fortran@v1
+      - name: Install MODFLOW 6
+        uses: modflowpy/install-modflow-action@v1
+        with:
+          repo: ${{ matrix.modflow6-repo }}
       - name: Install test dependencies
         run: |
           pip install -e ".[tests]"
-      - name: Test with pytest
-        # The flag -s stops pytest from capturing output
-        # This is necessary until proper error reporting is implemented by Modflow
+      - name: Run tests with MODFLOW 6 release
         run: |
-          pytest -vs --cov=xmipy --cov-report=xml tests/
+          pytest -v --cov=xmipy --cov-report=xml
```

Comment:
Why remove the `-s`?

MODFLOW 6 still crashes for most errors, which means there will be zero output in the CI when tests fail

Reply:
Running `pytest -s` throws way too much to the console, whereas the default will enable pytest to capture the output, showing more "normal" `pytests` outputs. I'll test locally to see if MF6 crashes are adequately captured with/without the flag...

Reply:
Let me know if you got reasonable results.

You could try e.g. an invalid configuration option.